vulkaninfo/vulkaninfo.cpp

```diff
@@ -210,14 +210,15 @@ void DumpPresentableSurfaces(Printer &p, AppInstance &inst, const std::vector<st
             auto exists = surface_list.end();
             for (auto it = surface_list.begin(); it != surface_list.end(); it++) {
                 // use custom comparator to check if the surface has the same values
```

Comment:
nit: I realize this comment wasn't modified by this PR, but is it a TODO, or meant to describe the following line?  Perhaps delete?

Reply:
Its a comment about why its comparing pointer's for the gpu's but comparing values for the surfaces. 
Since its not obvious at all why that is just from looking at the function, I felt it needed a comment.
But re-reading it, the language could be cleared up to not sound like a TODO

Reply:
ok.  using the term "comparator" lead me to a TODO for an operator==().  Perhaps "comparison"?  definitely a nit  :-)

Reply:
Nits are fine by me. I have made it more clear that its a comment, will push with the copyright changes.